t/t0005-rexec.t

```diff
@@ -17,8 +17,8 @@ test_expect_success 'basic rexec functionality (process success)' '
 	${FLUX_BUILD_DIR}/t/rexec/rexec /bin/true
 '
 
-test_must_fail 'basic rexec functionality (process fail)' '
-	${FLUX_BUILD_DIR}/t/rexec/rexec /bin/false
+test_expect_success 'basic rexec functionality (process fail)' '
+	! ${FLUX_BUILD_DIR}/t/rexec/rexec /bin/false
```

Comment:
technically you want `test_must_fail` here, or maybe better yet `test_expect_code 1 ...`

Not a showstopper for a merge though

Reply:
I saw this error message pop up. 

```
./sharness.sh: line 505: basic rexec functionality (process fail): command not found
test_must_fail: command not found: basic rexec functionality (process fail) 
	${FLUX_BUILD_DIR}/t/rexec/rexec /bin/false
```

It's when I realized, the mistake was I used `test_must_fail` as the primary uhhhh "expect" test  call (i.e. `test_expect_success`, `test_expect_failure`).  perhaps I should have done

```
test_expect_success 'basic rexec functionality (process fail)' '
	test_must_fail ${FLUX_BUILD_DIR}/t/rexec/rexec /bin/false
```

?

Reply:
Yes, that is the suggested use  of `test_must_fail` (and what I meant above. It is not suggested to use `!` to negate exit status because it would register success even if the program crashed) Like I said though, not a huge deal.

`test_expect_code 1 ..` would also work here.